python/modules/Stop0lObjectsProducer.py

```diff
@@ -133,21 +133,23 @@ def SelSoftb(self, isv):
 
 
     def SelJets(self, jet):
-        if jet.pt < 20 or math.fabs(jet.eta) > 2.4 :
+        if jet.pt < 20 or math.fabs(jet.eta) > 2.4:
             return False
         return True
 
     def SelPhotons(self, photon):
-        if photon.pt < 200:
+        if photon.pt < 220:
             return False
         abeta = math.fabs(photon.eta)
-        if (abeta > 1.442 and abeta < 1.566) or (abeta > 2.5):
+        if (abeta > 1.4442 and abeta < 1.566) or (abeta > 2.5):
```

Comment:
1.442 -> 1.4442? Is this a typo?

Reply:
Actually 1.4442 is what I have used for a while, and I am thinking 1.442 was incorrect. These are probably from a twiki...

Reply:
Hold on, can you find the source? The digit seems inconsistent

Reply:
I will investigate.... here is an old issue: https://github.com/susy2015/StopTupleMaker/issues/6

and here is the twiki: https://twiki.cern.ch/twiki/bin/view/CMS/CutBasedPhotonIdentificationRun2#Offline_selection_criteria

Here is what we used in 2018: https://github.com/susy2015/StopTupleMaker/blob/aae2e68dd08aacc5fe5f73a8ba48fb714f5a2525/SkimsAUX/plugins/PhotonIDisoProducer.cc#L278-L291

Reply:
So for our old ntuples we used 1.4442, 1.566, and 2.5 as the boundaries of the barrel and endcap.

Reply:
I don't know where these boundaries are documented though.

Reply:
https://github.com/TreeMaker/TreeMaker/blob/99a63a2ab17822bc2ef3b5ef8d06feec1d122429/Utils/src/PhotonIDisoProducer.cc#L251
I went in search of this number because I thought Andres and I had the 1.4442 and could not remember, but I found it in the RA2B code so I believe this is where it comes from.

Reply:
We see the values 1.4442 and 1.5660 used in CMSSW (which have the same precision).

https://github.com/cms-sw/cmssw/blob/02d4198c0b6615287fd88e9a8ff650aea994412e/DQM/Physics/python/singleTopDQM_miniAOD_cfi.py#L47